src/main/resources/assets/rebornstorage/lang/de_de.lang

```diff
@@ -0,0 +1,32 @@
+itemGroup.rebornstorage=Reborn Storage
+
+#Items Disks
+item.rebornstorage.storagecell.256k.name=256k Speicherzelle
```

Comment:
Not sure, shouldn't all these be connected with hyphens?

Reply:
The refined storage translation didn't use a hyphen too. For example: "1k Speicherzelle".